nightswatcher/nightswatcher.py

```diff
@@ -174,10 +177,29 @@ def extract_build(artifact_zip, build):
     tmp_artifact_path = tmp_version_dir + download_artifact
     if build['name'].startswith('build_android'):
         sign_apk(tmp_artifact_path)
-    shutil.move(tmp_artifact_path, download_artifact_path)
+    shutil.copy2(tmp_artifact_path, download_artifact_path)
+
+    ota_artifact_path = tmp_version_dir + download_artifact
+    # point update pointer to the right location
+    if build['name'] in ota_link_models:
+        if build['name'] == 'build_android_x86':
+            platform = platform + '-x86'
+
+        link_file_stable = OTA_DIR + ('koreader-%s-latest-stable' % platform)
+        link_file_nightly = OTA_DIR + ('koreader-%s-latest-nightly' % platform)
+        link_file = stable is True and link_file_stable or link_file_nightly
+
+        shutil.copy2(tmp_artifact_path, OTA_DIR)
```

Comment:
Actually we can just make a symlink here since it's a duplicate of the one in download, using [os.symlink](https://docs.python.org/3/library/os.html#os.symlink).